src/NServiceBus.Core/Routing/RoutingFeature.cs

```diff
@@ -31,28 +33,24 @@ protected internal override void Setup(FeatureConfigurationContext context)
             var transportInfrastructure = context.Settings.Get<TransportInfrastructure>();
 
             var unicastRoutingTable = context.Settings.Get<UnicastRoutingTable>();
+            unicastRoutingTable.SetLogChangeAction(GetLogAction<UnicastRoutingTable>());
```

Comment:
I could not figure out the other way of testing the logging. I know it is a pita but I think it is better than having an untested code, especially that our ATs run without debug logging enabled so this piece of code would never get executed.